spec/models/media_attachment_spec.rb

```diff
@@ -50,4 +49,12 @@
       expect(media.file.meta["small"]["aspect"]).to eq 400.0/267
     end
   end
+
+  describe 'descriptions for remote attachments' do
+    it 'are cut off at 140 characters' do
```

Comment:
are -> is

Reply:
descriptions for remote attachments -> are cut off at 140 characters

Reply:
yeah but 'it'...